packages/device_info_plus_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.3
```

Comment:
This isn't following semver; changing a return value from non-nullable to nullable is a breaking change.

Reply:
Yes, and also requires to release the main package and all the subpackages because the `device_info_plus_platform_interface` is going to be incompatible. Yay federated plugins 🤦 

There's no much you can do @KirillArtemenko we need to take it from here.

Reply:
Perhaps then you should temporarily use my previous solution https://github.com/fluttercommunity/plus_plugins/pull/257 ?